big_tests/tests/rest_helper.erl

```diff
@@ -316,9 +316,9 @@ is_roles_config(_, _) -> false.
 
 mapfromlist(L) ->
     Nl = lists:map(fun({K, {V}}) when is_list(V) ->
-                           {binary_to_existing_atom(K, utf8), mapfromlist(V)};
+                           {binary_to_atom(K, utf8), mapfromlist(V)};
```

Comment:
Why make that change now?

Reply:
it fails here, if run just one test.
K = subject.